### DISCUSSION TITLE
How to Delete a Folder in an Archive

### DISCUSSION THREAD

**Discussion by raywood1** [borgbackup/borg#6452] [Q&A] [answered]
I thought the syntax was something like this:

`sudo borg recreate /media/veracrypt1/BorgDriveBups::2022-01-01 --exclude "home/ray/borgmnt/media/2022-01-01/Long Folder Name"`

but that doesn't delete the folder. I've reviewed some discussions (notably [#141](https://github.com/borgbackup/borg/issues/141) and [#338](https://github.com/borgbackup/borg/issues/338)) but am unfortunately not finding a solution.

I used a mountpoint in home to avoid the “Mountpoint must be a writable directory” error.

**Comment by ThomasWaldmann** [comment, marked as answer]
is 2022-01-01 a valid archive name in that repo?

does the exclude match filenames in borg list output?

does this involve any kind of non-ascii characters?

**Reply by raywood1** [reply]
Apparently the solution was to run _borg list_. It showed that Borg had put the folder at a path that was not quite what I expected. Modifying the command to exclude that path seems to have worked. I will eventually post [a detailed writeup](https://raywoodcockslatest.wordpress.com/2022/02/18/borg-full-drive/#del). Thank you for your help.
